internal/v1/api.yaml

```diff
@@ -573,6 +573,26 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Filesystem'
+        users:
+          type: array
+          items:
+            $ref: '#/components/schemas/User'
+    User:
+      type: object
+      required:
+        - name
+      properties:
+        name:
+          type: string
+          example: "user1"
+        key:
```

Comment:
Should this be called `SSHKey` or something like that? Are we making sure we restrict what image types this applies to? I think this should only apply to installers, not to commits / base images. @gicmo thoughts?

Reply:
+1 on a 400 for anything other than the installer types.

Reply:
+1 on naming the sshkey too :)